src/concatkdf.c

```diff
@@ -7,12 +7,25 @@
 
 #include "include/concatkdf_int.h"
 
+
+#ifdef _WIN32
+#include <Winsock2.h>
+#include <malloc.h>
+#else
 #include <arpa/inet.h>
+#include <alloca.h>
+#endif
 #include <openssl/evp.h>
 #include <string.h>
 #include <cjose/base64.h>
 #include <cjose/util.h>
 
+#ifdef _WIN32
+#define StackAlloc _alloca
```

Comment:
I'd prefer to keep with the convention of single case; for macros that has been all upper case.